application/views/notes/main.php

```diff
@@ -1,38 +1,94 @@
-				{
-					echo "<h3>".__("Your Notes")."</h3>";
-					echo "<ul class=\"list-group\">";
-					foreach ($notes->result() as $row)
-					{
-						echo "<li class=\"list-group-item\">";
-						echo "<a href=\"".site_url()."/notes/view/".$row->id."\">".$row->title."</a>";
-						echo "</li>";
-					}
-					echo "</ul>";
-				} else {
-					echo "<p>".__("You don't currently have any notes, these are a fantastic way of storing data like ATU settings, beacons and general station notes and its better than paper as you can't lose them!")."</p>";
-				}
-
-			?>
-	  </div>
+		<div class="card-header">
+			<h2 class="card-title"><?= __("Notes"); ?></h2>
+			<ul class="nav nav-tabs card-header-tabs">
+				<li class="nav-item">
+					<a class="nav-link active" href="<?= site_url('notes'); ?>">
+						<i class="fa fa-sticky-note"></i> <?= __("Your Notes"); ?>
+					</a>
+				</li>
+				<li class="nav-item">
+					<a class="nav-link" href="<?= site_url('notes/add'); ?>">
+						<i class="fa fa-plus-square"></i> <?= __("Create Note"); ?>
+					</a>
+				</li>
+			</ul>
+		</div>
+		<div class="card-body">
+			<div class="row pt-2 align-items-center flex-wrap">
+				<div class="col-md-7 col-12 mb-2 mb-md-0">
+					<!-- Category filter buttons -->
+					<div id="categoryButtons" class="btn-group me-2 flex-wrap" role="group" aria-label="Category Filter">
+						<button type="button" class="btn btn-sm btn-outline-secondary btn-light category-btn active" data-category="__all__">
+							<?= __("All Categories"); ?> <span class="badge bg-secondary"><?= $all_notes_count ?></span>
+						</button>
+						<?php foreach ($categories as $cat): ?>
+							<button type="button" class="btn btn-sm btn-outline-secondary btn-light category-btn" data-category="<?= htmlspecialchars($cat) ?>">
+								<?= __(htmlspecialchars($cat)); ?>
+								<?php if ($cat === 'Contacts'): ?>
+									<span class="ms-1" data-bs-toggle="tooltip" title="<?= __("Contacts") . __(" is a special note category used in various parts of Wavelog to store information about QSO partners. These notes are private and are neither shared with other users nor exported to external services.") ?>">
+										<i class="fa fa-question-circle"></i>
+									</span>
+								<?php endif; ?>
+								<span class="badge bg-secondary"><?= $category_counts[$cat] ?? 0 ?></span>
+							</button>
+						<?php endforeach; ?>
+						<a href="<?php echo site_url('notes/add'); ?>" class="btn btn-sm btn-success">
+							<i class="fas fa-plus"></i> <?= __("Create Note"); ?>
+						</a>
+					</div>
+				</div>
+				<div class="col-md-5 col-12">
+					<!-- Search box and reset button -->
+					<div class="input-group">
+						<input type="text" id="notesSearchBox" class="form-control form-control-sm" maxlength="50" placeholder="<?= __('Search notes (min. 3 chars)') ?>">
+						<button class="btn btn-outline-secondary btn-sm btn-light" id="notesSearchReset" type="button" title="<?= __('Reset search') ?>">
```

Comment:
Please always use double quotes in translations

```suggestion
						<input type="text" id="notesSearchBox" class="form-control form-control-sm" maxlength="50" placeholder="<?= __("Search notes (min. 3 chars)") ?>">
						<button class="btn btn-outline-secondary btn-sm btn-light" id="notesSearchReset" type="button" title="<?= __("Reset search") ?>">
```